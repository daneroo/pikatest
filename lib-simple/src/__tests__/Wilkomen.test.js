import React from 'react'
import renderer from 'react-test-renderer'

import { Wilkomen } from '../index'

describe('Wilkomen', () => {
  test('import Wilkomen', () => {
    expect(typeof Wilkomen).toBe('function')
  })
  test('Wilkomen renders', () => {
    const component = renderer.create(
      <Wilkomen />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
