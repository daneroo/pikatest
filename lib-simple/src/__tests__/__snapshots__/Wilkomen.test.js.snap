// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wilkomen Wilkomen renders 1`] = `
<h4>
  Counting: 
  0
   from a component in a library with a hook
</h4>
`;
